lib_standard_app/Makefile.standard_app

```diff
@@ -0,0 +1,168 @@
+        STANDARD_APP_DEFINES += PRINTF=screen_printf
+    else
+        STANDARD_APP_DEFINES += PRINTF=mcu_usb_printf
+    endif
+else
+        STANDARD_APP_DEFINES += PRINTF\(...\)=
+endif
+
+#####################################################################
+#                        IO SEPROXY BUFFER SIZE                     #
+#####################################################################
+ifneq ($(DISABLE_DEFAULT_IO_SEPROXY_BUFFER_SIZE), 1)
+    ifeq ($(TARGET_NAME), TARGET_NANOS)
+        STANDARD_APP_DEFINES += IO_SEPROXYHAL_BUFFER_SIZE_B=128
+    else
+        STANDARD_APP_DEFINES += IO_SEPROXYHAL_BUFFER_SIZE_B=300
+    endif
+endif
+
+#####################################################################
+#                          STANDARD DEFINES                         #
+#####################################################################
+STANDARD_APP_DEFINES += $(DEFINES_LIB)
+STANDARD_APP_DEFINES += APPNAME=\"$(APPNAME)\"
+STANDARD_APP_DEFINES += APPVERSION=\"$(APPVERSION)\"
+STANDARD_APP_DEFINES += MAJOR_VERSION=$(APPVERSION_M) MINOR_VERSION=$(APPVERSION_N) PATCH_VERSION=$(APPVERSION_P)
+STANDARD_APP_DEFINES += OS_IO_SEPROXYHAL
+
+ifeq ($(DISABLE_STANDARD_APP_DEFINES), 1)
+    DISABLE_STANDARD_SNPRINTF = 1
+    DISABLE_STANDARD_USB = 1
+    DISABLE_STANDARD_BLE = 1
+    DISABLE_STANDARD_WEBUSB = 1
+    DISABLE_STANDARD_BAGL_UX_FLOW = 1
+endif
+
+ifneq ($(DISABLE_STANDARD_SNPRINTF), 1)
+    STANDARD_APP_DEFINES += HAVE_SPRINTF HAVE_SNPRINTF_FORMAT_U
+endif
+
+ifneq ($(DISABLE_STANDARD_USB), 1)
+    STANDARD_APP_DEFINES += HAVE_IO_USB HAVE_L4_USBLIB IO_USB_MAX_ENDPOINTS=6 IO_HID_EP_LENGTH=64 HAVE_USB_APDU
+    STANDARD_APP_DEFINES += USB_SEGMENT_SIZE=64
+endif
+
+ifneq ($(DISABLE_STANDARD_BLE), 1)
+    STANDARD_APP_DEFINES += BLE_SEGMENT_SIZE=32
+endif
+
+ifneq ($(DISABLE_STANDARD_WEBUSB), 1)
+    STANDARD_APP_DEFINES += HAVE_WEBUSB WEBUSB_URL_SIZE_B=0 WEBUSB_URL=""
+endif
+
+ifneq ($(DISABLE_STANDARD_BAGL_UX_FLOW), 1)
+ifneq ($(TARGET_NAME), TARGET_STAX)
+    STANDARD_APP_DEFINES += HAVE_UX_FLOW
+endif
+endif
+
+ifneq ($(DISABLE_STANDARD_APP_FILES), 1)
+SDK_SOURCE_PATH += lib_standard_app
+endif
+
+#####################################################################
+#                              NBGL                                 #
+#####################################################################
+ifeq ($(ENABLE_NBGL_QRCODE), 1)
+ifeq ($(TARGET_NAME), TARGET_STAX)
+    STANDARD_APP_DEFINES += NBGL_QRCODE
+endif
+endif
+
+ifeq ($(ENABLE_NBGL_KEYBOARD), 1)
+ifeq ($(TARGET_NAME), TARGET_STAX)
+    STANDARD_APP_DEFINES += NBGL_KEYBOARD
+endif
+endif
+
+ifeq ($(ENABLE_NBGL_KEYPAD), 1)
+ifeq ($(TARGET_NAME), TARGET_STAX)
+    STANDARD_APP_DEFINES += NBGL_KEYPAD
+endif
+endif
+
+CUSTOM_APP_FLAGS ?= 0x000
+
+APP_FLAGS = $(shell echo $$(( $(STANDARD_APP_FLAGS) + $(CUSTOM_APP_FLAGS) )) )
+
+APP_LOAD_PARAMS += --appFlags $(APP_FLAGS)
+
+APP_LOAD_PARAMS += $(COMMON_LOAD_PARAMS)
+
+DEFINES += $(STANDARD_APP_DEFINES)
```

Comment:
Maybe you could directly put stuff in `DEFINES` instead of having them in `STANDARD_APP_DEFINES` temporally?